Lib/sre_constants.py

```diff
@@ -21,6 +21,17 @@
 # should this really be here?
 
 class error(Exception):
+    '''Exception raised for invalid regular expressions.
```

Comment:
If this is only raised during compilation, then
"Exception raised while compiling invalid regular expressions.